src/components/TransactionHistoryRows/components/AssetsTransfers.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Circle, Stack, Text as CText } from '@chakra-ui/react'
+import { Amount } from 'components/Amount/Amount'
+import { useButtonStyles } from 'hooks/useButtonStyles/useButtonStyles'
+import type { Transfer } from 'hooks/useTxDetails/useTxDetails'
+import { bn, bnOrZero } from 'lib/bignumber/bignumber'
+import { fromBaseUnit } from 'lib/math'
+
+import { FALLBACK_PRECISION } from '../constants'
+
+type AssetsTransfersProps = {
+  compactMode?: boolean
+  transfers: Transfer[]
+  index: number
+}
+
+export const AssetsTransfers: React.FC<AssetsTransfersProps> = ({
+  index,
+  compactMode,
+  transfers,
+}) => {
+  const { bg: circleBgColor, color: circleColor } = useButtonStyles({
+    colorScheme: 'blue',
+    variant: 'ghost-filled',
+  })
+
+  const aggregatedFiatValue = transfers
+    .reduce(
+      (acc, transfer) =>
+        acc.plus(
+          bnOrZero(
+            fromBaseUnit(transfer.value, transfer.asset?.precision ?? FALLBACK_PRECISION),
+          ).times(transfer.marketData.price),
+        ),
+      bn(0),
+    )
+    .toString()
```

Comment:
more readable formatting imo (I moved `.toString()` down to component since that makes the formatting a bit more awkward, but you can keep if you like)
```suggestion
  const aggregatedFiatValue = transfers.reduce((prev, transfer) => {
    const precision = transfer.asset?.precision ?? FALLBACK_PRECISION
    const cryptoValue = fromBaseUnit(transfer.value, precision)
    return prev.plus(bnOrZero(cryptoValue).times(transfer.marketData.price))
  }, bn(0))
```

Reply:
also, probably best to memoize? maybe not as important since the `AssetTransfers` components are memoized in `TransactionRowGeneric`, but feels like it can just follow the memoize everything pattern?

Reply:
Re:suggestion, agree explicit over implicit is always worth the extra boilerplate.

Re: Memo, technically won't cause re-renders as it's a string primitive (it would as a BN), but yeah memoize everything is the way, no reason to recompute this if we don't need to! 2019d7ccc76e0711649b79e5967b368e387fde03